packages/babel-preset-react/src/index.js

```diff
@@ -5,15 +5,28 @@ import transformReactJSXSource from "@babel/plugin-transform-react-jsx-source";
 import transformReactJSXSelf from "@babel/plugin-transform-react-jsx-self";
 
 export default function(context, opts = {}) {
-  const development = opts.development || false;
+  let development = false;
+  let useBuiltIns = false;
+
+  if (opts !== undefined) {
+    if (opts.development !== undefined) development = opts.development;
+    if (opts.useBuiltIns !== undefined) useBuiltIns = opts.useBuiltIns;
+  }
 
   if (typeof development !== "boolean") {
-    throw new Error("Preset react 'development' option must be a boolean.");
+    throw new Error(
+      "@babel/preset-react 'development' option must be a boolean.",
+    );
+  }
+  if (typeof useBuiltIns !== "boolean") {
+    throw new Error(
+      "@babel/preset-react 'useBuiltIns' option must be a boolean.",
+    );
   }
 
   return {
     plugins: [
-      transformReactJSX,
+      [transformReactJSX, { useBuiltIns }],
       transformSyntaxJSX,
```

Comment:
It's confusing to me that the `syntax` plugin is coming _after_ the `transform` plugin

Reply:
yeah it shouldn't be there at all but I think someone wanted it anyway